app/models/commodity.rb

```diff
@@ -55,10 +55,6 @@ def to_s
     formatted_description || description
   end
 
-  def footnotes
```

Comment:
Hrm, looking at the previous codes it looks like ECO footnotes might sometimes _not_ be associated with the commodity but to a measure that is then associated with a commodity - so we should check both.